lib/project_types/script/commands/create.rb

```diff
@@ -29,10 +27,7 @@ def call(args, _name)
           no_config_ui: options.flags.key?(:no_config_ui)
         )
         @ctx.puts(@ctx.message("script.create.change_directory_notice", project.script_name))
-      rescue Script::Errors::ScriptProjectAlreadyExistsError => e
-        UI::ErrorHandler.pretty_print_and_raise(e, failed_op: @ctx.message("script.create.error.operation_failed"))
```

Comment:
This was here to prevent the cleanup on line 35 that would delete the existing script project. I moved this logic into the ScriptProjectRepository#create method as it should be responsible for cleaning up the files it made in the case of an error.

Reply:
Does it delete an existing project if the name conflicts?

Reply:
Nope! The folder will only be deleted if it was created during that specific command execution